src/encoded/tests/test_indexing.py

```diff
@@ -232,6 +232,7 @@ def test_file_processed_detailed(app, testapp, indexer_testapp, award, lab, file
     assert found_rel_sid > found_fp_sid  # sid of related file is greater
 
 
+@pytest.mark.skip(reason="validation_errors facet removed in search.py")
```

Comment:
I think this test should still work... Does it not?

Reply:
@willronchetti My bad, this test is not related to facets. I'll remove the decorator shortly.